.circleci/config.yml

```diff
@@ -214,6 +223,18 @@ commands:
     description: >-
       skip job unless required
     steps:
+      - run:
+          name: skip_job_from_enviroment_variable
```

Comment:
s/enviroment/environment/

Reply:
done

Reply:
This is still misspelled--should be "environment"